templates-generator/react-config.js

```diff
@@ -18,16 +18,16 @@ module.exports = {
             glob: 'src/app-routes.js',
             definitions: [
                 {
-                    before: 'import',
-                    after: '<%=^empty%>import'
+                    before: /import {([\s\w,]*)} from '.\/pages'/,
```

Comment:
To make regular expressions of the same type use "import {([^}]*)} from '.\/pages'"